src/python-frontend/python_converter.cpp

```diff
@@ -1001,10 +1026,18 @@ void python_converter::get_var_assign(
   {
     // Get type from current annotation node
     size_t type_size = 0;
-    if (
-      ast_node["value"].contains("value") &&
-      ast_node["value"]["value"].is_string())
-      type_size = ast_node["value"]["value"].get<std::string>().size();
+    if (ast_node["value"].contains("value"))
```

Comment:
Shall we consider adding some error handling here?

Reply:
I don't think so in this case. The information of type_size being obtained in this if statement is only necessary when dealing with arrays. If it's not an array, type_size can simply remain zero because it's not used in `get_typet`.